buildpacks/cnb.yaml

```diff
@@ -0,0 +1,54 @@
+---
+apiVersion: build.knative.dev/v1alpha1
+kind: BuildTemplate
+metadata:
+  name: cnb
+spec:
+  parameters:
```

Comment:
The old buildpack template supported a `CACHE` parameter, does buildpacks v3 support that today? It would be great if we could just change buildpack.yaml to run the new v3 images under the hood, without changing parameters at all.

Reply:
Definitely agree that we should keep this parameter so that an interbuild cache volume can be used.

There are two reasons we have separate templates with separate arguments for the CF and CNB models:
- The models are very different. CNB:
  - requires a separate analyze step to pull metadata from the previous image
  - requires detection
  - requires a more complex buildpack order definition
- The included buildpacks are different and incompatible. The CF template uses the Cloud Foundry buildpacks (by retrofitting the Cloud Foundry lifecycle into builder images). The CNB template currently includes [these samples](https://github.com/buildpack/samples) and may eventually include function invoker buildpacks.